### PR TITLE
docs: Create SECURITY.md

### PR DESCRIPTION
**Pull request by lirantal**
Adding a basic security policy

This can later be extended with responsible disclosure guidelines, time frames, etc.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.x     | :white_check_mark: |
```
Comment (lirantal) [inline SECURITY.md:7]:
```suggestion
| 1.x     | :x: |
```

Comment (lirantal) [reply SECURITY.md:7]: commit update to address 1.x lack of support

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by lirantal** [comment]
All good :-)

**Review by gurgunday — APPROVED** [review]

**Pull request merged**